dll/win32/ieframe/ieframe_v1.rgs

```diff
@@ -5,9 +5,21 @@ HKCR
     }
     NoRemove CLSID
     {
-        '{871C5380-42A0-1069-A2EA-08002B30309D}' = s 'Internet'
+        '{871C5380-42A0-1069-A2EA-08002B30309D}' = s 'Internet Browser'
```

Comment:
why that change?

Reply:
This is for consistency with desk.cpl option.

Reply:
The international settings are not correctly applied on Desktop icon text.

Reply:
We could also use just "Internet" in the dek.cpl to harmonize. Would even make the translation easier, and "Internet" is more an internationalism than "Internet Browser" imho. So it may avoid the need for translation for some languages entirely. I would volunteer personally to harmonize desk.cpl later.

Reply:
Windows desk.cpl uses two words "Internet Explorer", using one word there wouldn't look consistent. I would not prefer that UX decision.

Reply:
That is no argument at all, because "Internet Browser" is also not the same as "Internet Explorer".

Reply:
There are 2 words in both cases, it looks consistent from the UI point of view. Using just one word there would make it an oddball. Also using just "Internet" is quite ambiguous. These are two reasons I have chosen "Internet Browser" when I worked on desktop icons settings in desk.cpl.

Reply:
Let's see whether there are more upvotes for your or for my position ultimately and then stick with the decision of the majority. Let the thumbs decide. My current rejection of this PR is not due to this specific discussion-thread.

Reply:
https://chat.reactos.org/reactos/pl/ysjmbfeempff9pwd3ofwbzaz8r

Reply:
Voting: https://chat.reactos.org/reactos/pl/woka83o84jbnuy1m33r7kqeo4r

Reply:
Charged to `Internet`.

Reply:
The poll in the chat has been started in the time when I was not online, and the change has been pushed earlier than 24 hours from the poll start, this is illegal.

Reply:
You can start a poll it again if you want.

Reply:
Reverted to `"Internet Browser"`.